### PR TITLE
Update headings in Checkout

### PR DESCRIPTION
**Pull request by gustaveen**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2165** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2165" target="_blank">Graphite</a> 
    * **PR #2180** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2180" target="_blank">Graphite</a> 
      * **PR #2183** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2183" target="_blank">Graphite</a> 
        * **PR #2184** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2184" target="_blank">Graphite</a> 
          * **PR #2186** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2186" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2186?utm_source=stack-comment).

File: apps/store/src/components/CheckoutPage/CheckoutPage.tsx [hunk 1]
```diff
@@ -100,10 +100,15 @@ const CheckoutPage = (props: CheckoutPageProps) => {
           <TextLink href={PageLink.cart()}>{t('BACK_BUTTON')}</TextLink>
         </CheckoutHeader>
         <Layout>
-          <Content y={{ base: 2.5 }}>
-            <Heading balance as="h1" variant="standard.24" align="center">
```
Comment (robinandeer) [inline apps/store/src/components/CheckoutPage/CheckoutPage.tsx:104]: Is the text balancer not working for us?

**Review by robinandeer — APPROVED** [review]
Code looks good

File: apps/store/src/components/CheckoutPage/CheckoutPage.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/components/CheckoutPage/CheckoutPage.tsx:104]: It is! In this case the new title is just "Sign" so I thought we didn't need it in this case. :)

**Comment by gustaveen** [comment]
[Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2186) rebased this pull request as part of a merge.

**Comment by gustaveen** [comment]
@gustaveen queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2186).

**Pull request merged**